Improve docs by removing typos and grammatical errors (#28391)
github/docs: content/codespaces/managing-codespaces-for-your-organization/choosing-who-owns-and-pays-for-codespaces-in-your-organization.md

---
title: Choosing who owns and pays for codespaces in your organization

> shortTitle: 'Billing and Ownership'

Reviewer: We use sentence casing for titles short titles, so the original is correct.

```suggestion
shortTitle: 'Billing and ownership'
```

intro: 'You can choose whether codespaces are paid for and owned by your organization or by your members.'

> permissions: 'Organization owners can change an organization's billing details and control who owns and pays for codespaces.'

Reviewer: The first apostrophe is required as an escape, and removing it is causing the test failure in your PR. We can switch to double quotes to make the source more readable by preventing the need for the escape.

```suggestion
permissions: "Organization owners can change an organization's billing details and control who owns and pays for codespaces."
```

  - Codespaces
  - Billing
  - Administrator

> product: 'Organizations on {% data variables.product.prodname_team %} and {% data variables.product.prodname_enterprise %} plans can pay for their members' use of {% data variables.product.prodname_github_codespaces %}. These organizations can then access policies that apply to codespaces paid for by the organization. For more information, see "[AUTOTITLE](/get-started/learning-about-github/githubs-products)."'

Reviewer: Same as above, except in this case switching to double quotes doesn't really help, as we'd just need to escape the double quotes around the link.

```suggestion
product: 'Organizations on {% data variables.product.prodname_team %} and {% data variables.product.prodname_enterprise %} plans can pay for their members'' use of {% data variables.product.prodname_github_codespaces %}. These organizations can then access policies that apply to codespaces paid for by the organization. For more information, see "[AUTOTITLE](/get-started/learning-about-github/githubs-products)."'
```

---

## Overview

If you're the owner of an organization on a {% data variables.product.prodname_team %} or {% data variables.product.prodname_ghe_cloud %} plan, you can pay for your members' and collaborators' usage of {% data variables.product.prodname_github_codespaces %}. Paying for usage will allow people to use {% data variables.product.prodname_github_codespaces %} to work in your repositories without having to do so at their own expense and will give your organization more control over the codespaces created from your repositories.

To pay for usage, you must do all of the following things:

> - Allow at least some of your members and collaborators to use {% data variables.product.prodname_github_codespaces %} in your organization's private {% ifversion ghec %}and internal{% endif %} repositories. For more information, see "[AUTOTITLE](/codespaces/managing-codespaces-for-your-organization/enabling-or-disabling-github-codespaces-for-your-organization#enabling-or-disabling-github-codespaces)."

Reviewer: With your change, there are now two spaces between private and repositories for the non-GHEC version. The original is correct.

```suggestion
- Allow at least some of your members and collaborators to use {% data variables.product.prodname_github_codespaces %} in your organization's private {% ifversion ghec %}and internal {% endif %}repositories. For more information, see "[AUTOTITLE](/codespaces/managing-codespaces-for-your-organization/enabling-or-disabling-github-codespaces-for-your-organization#enabling-or-disabling-github-codespaces)."
```

If your organization owns a codespace, your organization has control over that codespace. For example, for codespaces owned by your organization, you can:

> - Use the [REST API](/rest/codespaces/organizations) to manage codespaces, such as stopping or deleting a codespace.
> - Access audit logs to review actions related to {% data variables.product.prodname_github_codespaces %}.
> - Set policies to manage constraints, such as restricting the base image or machine type that can be used in codespaces, or setting a default timeout and retention period.

Reviewer: We only use periods at the end of list items when each list item is a full sentence. In this case, each list item is a continuation of the introductory sentence rather than a sentence of its own.

```suggestion
- Use the [REST API](/rest/codespaces/organizations) to manage codespaces, such as stopping or deleting a codespace
- Access audit logs to review actions related to {% data variables.product.prodname_github_codespaces %}
- Set policies to manage constraints, such as restricting the base image or machine type that can be used in codespaces, or setting a default timeout and retention period
```
